testdata/output/telex__simple_telex.snap

```diff
@@ -38,7 +38,7 @@ daasd
 aa
 dd
 chuw
-chw
+chww
```

Comment:
This should not change tho. `chww` should be transformed as `chư` then `chw`

Reply:
IIUC unikey transforms `chư + w => chuw`.
I think transforming `chư + w` into `chw` instead of just `chww` is special casing too much.
Let me know what you think.

Reply:
I'm a VNI user so my thoughts would matter that much unfortunately 😂 @huytd could you help us with this since you're a telex user as I remember? 🤔

Reply:
May I suggest

1. Press `c` & `h` -> `ch`
2. Case 1: Press `w` -> `chư`
   1. Press `w` again -> `chw`.
   2. Press `w` one more time -> `chww`
3. Case 2: Press `u` -> `chu`
   1. Press `w` -> `chư`
   2. Press `w` again -> `chuw`. User would like to have `u` in the word.

Reply:
OK. A telex user seems to agree with the behaviour that you mentioned. We'll go with that behaviour for now until people complains 😄

Reply:
Since we don't have enough feedback & logical reasons. We'll go with the behaviour of @tesuji for now until the telex users start complaining.